cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -488,10 +489,12 @@ func newByAccountRegistersFromPayloadAccountGrouping(
 		for accountRegisters := range results {
 			oldAccountRegisters := registersByAccount.SetAccountRegisters(accountRegisters)
 			if oldAccountRegisters != nil {
-				return fmt.Errorf(
-					"duplicate account registers for account %s",
-					accountRegisters.Owner(),
-				)
+				log.Warn().Msgf("account registers already exist for %x, merging", accountRegisters.Owner())
+
+				err := accountRegisters.Merge(oldAccountRegisters)
```

Comment:
Could we add some comments here to explain why we decide to merge in this case?

Reply:
Done in [9adca6c](https://github.com/onflow/flow-go/pull/5942/commits/9adca6cf6eefe44e6918cbcc17164fe324e2d04a)